Wireframe/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  background-color: lavender;
+}
+
+.hero-image {
+  position: relative;
+  width: 100%;
+  height: 300px; /* Adjust height as needed */
+  overflow: hidden;
+}
+
+.hero-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover; /* Ensures the image covers the entire area */
+}
+
+.hero-text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  color: navy;
+}
+
+.text-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px;
+}
+
+main article {
```

Comment:
how might we select the first article to style it uniquely? (hint: the approach is mentioned in the requirements)